codegen/masm/src/lower/component.rs

```diff
@@ -43,7 +45,18 @@ impl ToMasmComponent for builtin::Component {
                 let name = masm::ProcedureName::from_raw_parts(masm::Ident::from_raw_parts(
                     Span::new(entry_id.function.span, entry_id.function.as_str().into()),
                 ));
-                let path = component_path.clone().append_unchecked(entry_id.module);
+
+                let path = if component_path.to_string() == "root_ns:root@1.0.0" {
```

Comment:
@bitwalker I fixed the "undefined module" error with this quick hack, but I feel there should be a better way. Maybe mark the synthetic component with an attribute so we can recognize it without matching the ID?